pandas/core/tools/datetimes.py

```diff
@@ -787,10 +787,6 @@ def to_datetime(
         - :const:`"%f"` will parse all the way
           up to nanoseconds;
 
-        - :const:`"%S"` without :const:`"%f"` will capture all the way
-          up to nanoseconds if present as decimal places, and will also handle
-          the case where the number of seconds is an integer.
-
```

Comment:
Now, the preceding paragraph says "though note the following difference", but then there's only one difference.

Perhaps we can change it to something like
```
though note that "%f" will parse all the way up to nanoseconds
```